src/index.js

```diff
@@ -94,10 +94,14 @@ class Radar {
       throw new Error(ERROR.PARAMETERS);
     }
 
-    Track.trackOnce(position)
+    Track.trackOnce(location)
       .then((response) => {
-        const { location, user, events } = response;
-        callback(null, location, user, events, response);
+        callback(null, {
```

Comment:
do we want the raw `response` to be part of the object we send back? might be odd to see from the caller's perspective if someone does `Radar.getContext((res) => console.log(res));` and sees double the information - we could keep the full response as the 3rd arg always, since the 2nd will always be an object

Reply:
That's true. The other thing we could do is include `meta` on on this object. We're doing anything _yet_ with meta, but it might be useful to surface it (eg. if we want to support pagination)

Reply:
On second thought, I wonder if we should just return the response object. It will just be the top-level keys + meta anyways.

Reply:
am fine with either approach - technically we're not consistent with this everywhere, as we don't destructure the individual components of the context response (geofences, place, regions), so I'm fine keeping it generic to return the entire response for now, and building in conveniences later